api/internal/handler/system_config/system_config_test.go

```diff
@@ -0,0 +1,258 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package system_config
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/shiningrush/droplet"
+	"github.com/shiningrush/droplet/data"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/apisix/manager-api/internal/core/entity"
+	"github.com/apisix/manager-api/internal/core/store"
+)
+
+func TestSystem_Get(t *testing.T) {
```

Comment:
In fact, we can remove these meaningless test cases, just keep only the E2E test.

Reply:
I think the unit test is also required, it can quickly detect coding problems

Reply:
https://github.com/apache/apisix-dashboard/blob/561ed377ec2237707bb8c78623e336360c6c6463/api/internal/handler/stream_route/stream_route_test.go#L31-L71

You can take a look at these, I don't think it makes sense to run these tests through Mock, but unit tests can be used such as data unmarshal.

Reply:
Do you mean `get` needs to test data unmarshal,  the other operations remain unchanged?

Reply:
I mean, the current use cases like `Get` `Put` use mock storage to test, which doesn't make a lot of sense because mock doesn't have its own storage features like in-memory, etc (We also don't know if the handler is actually working properly, because the input and output are provided by the developer code). The storage layer that is really being used will be tested in full in E2E, in unit we just do some other trivial work.

This is my own idea, what do you think?

Reply:
thx for your reply!

I think that only the data layer would use mock instead, and the unit test can walk through the handle logic, so it makes sense.

Maybe it's because the handle very simple and you think it doesn't make sense？ if that's the case, I take your advice.

Reply:
😂, I think all can, please @nic-chen @starsz to see it.

Reply:
Yes, most unit test cases are the same and meaningless, you could have a look at unit test cases of other handlers, they are almost the same.
We should focus on tests that are not covered.

Reply:
> and the unit test can walk through the handling logic, so it makes sense.

I think we can keep it, even if the logic is very simple and doesn't make sense.
Once the handle contains complex logic, the effect will be obvious.